src/matlab.ts

```diff
@@ -54,7 +57,7 @@ export async function runCommand(hs: HelperScript, platform: string, architectur
     const rmcPath = getRunMATLABCommandScriptPath(platform, architecture);
     await fs.chmod(rmcPath, 0o777);
 
-    const rmcArg = script.cdAndCall(hs.dir, hs.command);
+    const rmcArg = `setenv('MW_ORIG_WORKING_FOLDER', cd('${hs.dir}'));${hs.command}`;
```

Comment:
Wouldn't that mean that the other function consumers would likely have to escape the dir themselves later on? I can't think of many cases where someone would be calling `generateScript` and not wanting the directory to be escaped but i'm probably missing something

Reply:
`generateScript` just generates a script and returns the directory that script lives in. It shouldn't assume you're then going to inject that directory path into a MATLAB char array later on. If, for instance, you wanted to delete that directory from JavaScript code, you've got an issue because the directory path is wrong.